src/umpire/CMakeLists.txt

```diff
@@ -59,6 +59,21 @@ set(UMPIRE_ENABLE_LOGGING ${ENABLE_LOGGING})
 set(UMPIRE_ENABLE_SLIC ${ENABLE_SLIC})
 set(UMPIRE_ENABLE_ASSERTS ${ENABLE_ASSERTS})
 set(UMPIRE_ENABLE_STATISTICS ${ENABLE_STATISTICS})
+set(UMPIRE_ENABLE_ROCM ${ENABLE_ROCM})
```

Comment:
Did a description of this new option get added to https://github.com/LLNL/Umpire/blob/develop/docs/sphinx/advanced_configuration.rst?

Reply:
Nope 😞 I'll do that!